pkg/vif/tunnel_vif.go

```diff
@@ -0,0 +1,41 @@
+package vif
+
+import (
+	"context"
+
+	"gvisor.dev/gvisor/pkg/tcpip/stack"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/tunnel"
+	"github.com/telepresenceio/telepresence/v2/pkg/vif/device"
+	"github.com/telepresenceio/telepresence/v2/pkg/vif/netstack"
+)
+
+type TunnelVIF struct {
```

Comment:
The name `vif.TunnelVIF` repeats the term `vif`. Suggest using `vif.Tunnel` or even `vif.DeviceWithStack`.